instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/servlet/AppServerBridge.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.servlet;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Helper containers for the Context attributes for transferring certain information between servlet
+ * integration and app-server server handler integrations.
+ */
+public class AppServerBridge {
+
+  public static final ContextKey<AtomicReference<Throwable>> THROWABLE_CONTEXT_KEY =
```

Comment:
It's better if we can have a single key / object with multiple fields for any particular concern, rather than multiple keys

Reply:
Fixed.